vnext/include/ReactUWP/Views/ViewManagerBase.h

```diff
@@ -89,6 +89,8 @@ class REACTWINDOWS_EXPORT ViewManagerBase
 
   virtual void TransferProperties(XamlView oldView, XamlView newView);
 
+  virtual void OnDescendantTextPropertyChanged(ShadowNodeBase *node){};
```

Comment:
I don't think this should be on ViewManagerBase. It belongs on TextViewManager only right? You should just cast the parent's viewmanager  in the implementation to a TextViewManager.